contributors/design-proposals/storage/volume-provisioning.md

```diff
@@ -184,7 +184,7 @@ Existing behavior is unchanged for claims that do not specify
   3. It tries to find a StorageClass instance referenced by annotation
      `claim.Annotations["volume.beta.kubernetes.io/storage-class"]`. If not
      found, it SHOULD report an error (by sending an event to the claim) and it
-     SHOULD retry periodically with step i.
+     SHOULD retry periodically with step 1.
```

Comment:
I think step `i` is correct here, because it should retry with step 1 in the subpoints (where "1" is rendered as `i`).

To be clear, by step `i`, I mean this one:


> The provisioner inspects if claim.Annotations["volume.beta.kubernetes.io/storage-provisioner"] == <provisioner name>. All other claims MUST be ignored.

Reply:
Yea, let's close this "typo"

Reply:
@liujun-daocloud can you remove this change and squash the commits after that? Lgtm after that.